protos/core.proto

```diff
@@ -424,4 +429,12 @@ message TableMetadata {
   Target target = 1;
   string type = 2; // "table" or "view"
   repeated Field fields = 3;
+  int64 lastUpdatedMillis = 4;
+}
+
+message PersistedTableMetadata {
```

Comment:
We need to store the action name in here as a string right?

Reply:
While you don't need it in this PR, you will definitely need it to write updates correctly in the future!

Reply:
I added the target object, which is actually resolved to action name.